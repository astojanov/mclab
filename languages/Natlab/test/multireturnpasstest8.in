
[a.b, c(18), d(4).e] = foo();
