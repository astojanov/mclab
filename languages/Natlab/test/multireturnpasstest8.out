
[t1, t2, t3] = foo();
a.b = t1;
c(18) = t2;
d(4).e = t3;
